packages/pirania/files/usr/lib/lua/voucher/utils.lua

```diff
@@ -23,7 +23,7 @@ local function escapeCSV (s)
 end
 
 local function dateNow ()
-  return os.time() * 1000
```

Comment:
Might this affect the lime-app pirania integration? It looks that you are changing units, and the interface might depend on it...

Reply:
Yes,  but current code is broken as far as I understand (int32 overflow in the voucher logic code)

Reply:
Or I can comment/delete the tests and leave the code as is.

Reply:
Ok, please be aware of this change @luandro